libs/common/src/lib/navbar/navbar.component.ts

```diff
@@ -23,7 +23,7 @@ export class NavbarComponent implements OnInit, OnDestroy {
   );
   inSettings: Observable<boolean> = combineLatest([this.sdk.currentAccount, this.sdk.currentKb]).pipe(
     map(([account, kb]) => {
-      return this.navigationService.getKbUrl(account.slug, kb.slug || '');
+      return this.navigationService.getKbUrl(account.slug, this.standalone ? kb.id : kb.slug || kb.id);
```

Comment:
you have `this.standalone ? kb.id : kb.slug || kb.id` twice in the code, maybe it would worth extracting it in its own property?